src/main/java/mate/academy/springboot/datajpa/dto/request/ProductRequestDto.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.springboot.datajpa.dto.request;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
```

Comment:
don't use @Data without needing
https://thorben-janssen.com/lombok-hibernate-how-to-avoid-common-pitfalls/